tpm2_pytss/constants.py

```diff
@@ -316,6 +316,14 @@ def __str__(self):
         return "|".join(ints)
 
 
+@TPM_FRIENDLY_INT._fix_const_type
+class TPM2_HANDLE(TPM_FRIENDLY_INT):
+    """TPM2_HANDLE is a raw handle type.
+    """
+
+    pass
+
+
```

Comment:
TPM2_HANDLE is already defined in types.py

Reply:
Oh I was looking in constants.